cmd/agent/app/builder.go

```diff
@@ -231,7 +231,7 @@ func CreateCollectorProxy(
 	}
 	switch opts.ReporterType {
 	case reporter.GRPC:
-		return grpc.NewCollectorProxy(grpcBuilder, opts.AgentTags, mFactory, logger)
+		return grpc.NewCollectorProxy(grpcBuilder, opts.AgentTags, opts.DuplicateTagsPolicy, mFactory, logger)
```

Comment:
This function signature is getting too long, can we pass the opts object?

Reply:
I believe the pattern is to use an object-specific struct, like:

https://github.com/jaegertracing/jaeger/blob/dbfeb7abd5f3add174bce636a174b5ab2dcac4ea/cmd/collector/app/span_processor.go#L31-L40